generate/swagger.json

```diff
@@ -131,6 +171,154 @@
       },
       "type": "object"
     },
+    "resourceplugin.Runner": {
```

Comment:
I see there is no reference at all to the `basePodTemplate`... will that affect the `sigandot CLI`? will we be able to submit a plugin with a `basePodTemplate` in it?

Reply:
In this first step, I'm just enabling submitting the resource plugin void of basePodTemplate (as discussed on Slack yesterday). But that's the very first next thing once I get the skeleton out swagger changes in Signadot repo all the way to the CLI repo.

The reason for trying to get something sooner is to make sure that we have something to build on (though the absence of basePodTemplate makes it incomplete), and get stuck less in having to rebase as newer changes get merged.

Reply:
Created a ticket to track this: https://github.com/signadot/signadot/issues/2984